ddclient

```diff
@@ -303,6 +304,7 @@ sub ip_strategies_usage {
 my %web_strategies = (
```

Comment:
As far as I can tell `%web_strategies` is not used anywhere. Maybe delete it?

Reply:
You are right. Ddclient is just the definition of legacy code :)